### PR TITLE
Live push: Accept Dockerfile ENV instructions in push to local device

### PR DESCRIPTION
**Pull request by pdcastro**
Update livepush, chokidar and @types/dockerode dependencies.

Change-type: minor

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -113,10 +113,9 @@
     "@types/body-parser": "^1.19.0",
     "@types/chai": "^4.2.11",
     "@types/chai-as-promised": "^7.1.1",
-    "@types/chokidar": "^2.1.3",
```
Comment (pdcastro) [inline package.json:116]: Removed as advised at:
> https://www.npmjs.com/package/@types/chokidar
> This package has been deprecated
> Author message:
> This is a stub types definition. chokidar provides its own type definitions, so you do not need this installed.

File: tests/test-data/pkg/expected-warnings-darwin.txt [hunk 2]
```diff
@@ -68,6 +64,10 @@
   The addon must be distributed with executable as %2.
   %1: node_modules/fsevents/fsevents.node
   %2: path-to-executable/fsevents.node
+> Warning Cannot include addon %1 into executable.
+  The addon must be distributed with executable as %2.
+  %1: node_modules/xxhash/build/Release/hash.node
+  %2: path-to-executable/hash.node
```
Comment (pdcastro) [inline tests/test-data/pkg/expected-warnings-darwin.txt:70]: Same contents, in different order. To do: Improve the `pkg` diff comparison to accept same contents in different order.

File: patches/unix/node-gyp-build+4.2.3.patch [hunk 3]
```diff
@@ -1,5 +1,5 @@
 diff --git a/node_modules/node-gyp-build/index.js b/node_modules/node-gyp-build/index.js
-index 3e39703..5e84906 100644
+index b5096ed..7cd451a 100644
 --- a/node_modules/node-gyp-build/index.js
 +++ b/node_modules/node-gyp-build/index.js
```
Comment (pdcastro) [inline patches/unix/node-gyp-build+4.2.3.patch:4]: Apparently `node-gyp-build` had been updated in some previous PR that got merged, without the patch file being updated. Updating it here -- incidental to this PR.

File: package.json [hunk 4]
```diff
@@ -202,7 +201,7 @@
     "body-parser": "^1.19.0",
     "capitano": "^1.9.2",
     "chalk": "^3.0.0",
-    "chokidar": "^2.1.8",
+    "chokidar": "^3.3.1",
```
Comment (pdcastro) [inline package.json:204]: Set to `^3.3.1` to match livepush's `package.json`. What actually got installed and recorded in `npm-shrinkwrap` is `v3.4.2`.

File: tests/test-data/pkg/expected-warnings-darwin.txt (hunk 2 above)
Comment (Page-) [reply tests/test-data/pkg/expected-warnings-darwin.txt:70]: Maybe just a list of the paths, ie the `node_modules/xxhash/build/Release/hash.node` bits?

**Pull request merged**